docs/mesos/README.md

```diff
@@ -7,56 +7,58 @@
 
-## 1. Prepare Master and Agent Nodes
-The [Mesos Host Preparation tutorial](PrepareHosts.md) will walk you through hostname and firewall configuration for compatability between Calico and Mesos.
+###  Launching Tasks with Calico + Mesos
+IP-Per-Container Networking with Calico is an opt-in feature for Mesos Frameworks that launch tasks with [Networkinfo](https://github.com/apache/mesos/blob/0.26.0-rc3/include/mesos/mesos.proto#L1383). This means that your favorite Mesos Frameworks will not work with Calico until they have opted to include Networkinfo when launching tasks. Currently, this is limited to Mesos tasks launched via Marathon, with support for more frameworks growing. 
 
-## 2. Prepare Core Services
-Zookeeper and etcd serve as the backend datastores for Mesos and Calico, respectively. The [Core Services Preparation tutorial](PrepareCoreServices.md) will walk you through setting up both services using Docker.
+Since the Mesos Docker Containerizer does not support Module hooks, external networking is incompatible with docker containers in 0.26. Modifications are being made to the Mesos Containerizer to launch docker containers in future versions of Mesos, which will work with Calico out of the box going forward.
 
-## 3. Calico
-The following tutorials will help you install the Calico services required for each agent in your Mesos cluster.
+## 1. Prepare Master Node
+The [Mesos Master Preparation tutorial](MesosMasterPreparation.md) 
+will walk you through firewall configuration for compatability 
+between Calico and Mesos.
 
-### Calico With Docker (Recommended)
-Calico is primarily distributed as a Docker container. Follow one of the tutorials below to set that up.
+Zookeeper and etcd serve as the backend datastores for Mesos and 
+Calico, respectively. This tutorial also provides a quick start 
+guide to walk you through setting up both services using Docker.
 
-a.) [Install the Calico Networking in Mesos from RPM](RpmInstallCalico.md)
+## 2. Install
+Choose one of the following guides to install Mesos with Calico:
 
-b.) [Manually Install Calico Networking in Mesos](ManualInstallCalico.md)
+### a.) RPM
+The [Calico-Mesos RPM Installation Guide](RpmInstallCalicoMesos.md) serves as the fastest way to get up and running, by installing Mesos, Netmodules, and Calico onto your system.
 
-### Calico Without Docker
-Don't want to use Docker in your Mesos Cluster? Calico can run directly on your Agent. Choose one of the following tutorials to install Calico without Docker.
+### b.) Manual
+For an in-depth walkthrough of the full installation procedure performed by the RPMs, see the [Calico-Mesos Manual Install Guide](ManualInstallCalicoMesos.md).
 
-c.) Create and Install the Dockerless Calico-Mesos RPM (coming soon)
+### c.) Dockerized Demo
+Not interested in provisioning your own system? Folow the dockerized [net-modules demo][net-modules] to see how it all works.
```

Comment:
The Dockerized Demo is actually something different than the Net-modules demo. I am moving our docs away from the docker-compose demo, since its purpose is only to showcase what netmodules can do. Therefore, keep the title of this secino, but change the contents of c.) to something like this:

For those of you that don't want to install mesos onto your hosts, we've dockerized the important pieces, with Calico already set up. See the [Dockerized Mesos Guide](DockerizedDeployment.md) for info on how to get up and running, fast.